### PR TITLE
Business project method headers and datafields added to classes

### PR DESCRIPTION
**Pull request by AndreasBoegh**

### CODE REVIEW

File: HospitalGame_Business/src/business/Data/DataNPC.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Data;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class DataNPC {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Data/DataNPC.java]: Need to implement INPC

File: HospitalGame_Business/src/business/Data/DataPlayer.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Data;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class DataPlayer {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Data/DataPlayer.java]: Needs to implement IPlayer

File: HospitalGame_Business/src/business/Item/BloodBag.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Item;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class BloodBag {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Item/BloodBag.java]: Needs to implement IBloodBag, IItem, IBonusPointItem

File: HospitalGame_Business/src/business/Item/BonusPointItem.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Item;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class BonusPointItem {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Item/BonusPointItem.java]: Needs to implement IItem, IBonusPointItem

File: HospitalGame_Business/src/business/Item/Inventory.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Item;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class Inventory {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Item/Inventory.java]: Need to implement IInventory

File: HospitalGame_Business/src/business/Item/Item.java [hunk 6]
```diff
@@ -3,12 +3,12 @@
...
-package business;
+package business.Item;
 
 /**
  *
- * @author fsr19
+ * @author andreasmolgaard-andersen
  */
-public class NewClass {
+public class Item {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Item/Item.java]: Need to implement IItem

File: HospitalGame_Business/src/business/Item/ItemFacade.java [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Item;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class ItemFacade {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Item/ItemFacade.java]: Need to implement IItemFacade

File: HospitalGame_Business/src/business/Item/PowerUpItem.java [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package business.Item;
+
+/**
+ *
+ * @author andreasmolgaard-andersen
+ */
+public class PowerUpItem {
```
Comment (frederikrosenberg) [inline HospitalGame_Business/src/business/Item/PowerUpItem.java]: Need to implement IPowerUpItem, IItem

**Review by frederikrosenberg — CHANGES_REQUESTED** [review]
A lot of implements

**Review by frederikrosenberg — APPROVED** [review]

**Review by larsbjn — APPROVED** [review]

**Pull request merged**